src/stores/RegionStore.js

```diff
@@ -121,7 +124,6 @@ const SelectionMap = types.model(
     highlight(region) {
       self.clear();
       self.select(region);
-      region?.shapeRef?.parent?.canvas?._canvas?.scrollIntoView?.();
```

Comment:
why do you remove it? it was helpful for multiple images in config to scroll to the respective image when user selects some region.